services/api/Dockerfile-webserver

```diff
@@ -7,10 +7,10 @@ RUN npm clean-install
 COPY src/unified_graphics/static/ src/unified_graphics/static/
 RUN npm run vendor && npm run build
 
-FROM nginx:1.21
+FROM nginxinc/nginx-unprivileged:1.24
 
 COPY --from=build /home/node/app/build /usr/share/nginx/html/static
 COPY default.conf.template /etc/nginx/templates/
-RUN apt-get update \
-    && apt-get upgrade -y \
-    && rm -rf /var/lib/apt/lists/*
```

Comment:
We no longer need to update our packages for security scans?

Reply:
It's a tradeoff of using the `nginx-unprivileged` image. We don't have access to the root user in that image so can't run `apt update && upgrade`. I've been going back and forth on if this will be okay.

Nginx releases new patched container images for `nginx-unprivileged` every Monday. However, I could also use the original `nginx` image and drop permissions. We'd need to add more nginx config to the repo in that case but we would have more control in the image.

Reply:
As long as our deploys don’t get blocked by the security scans, it doesn’t matter to me.

Reply:
Sounds good. I'll give this one a go for a bit and if it causes us security problems, we can switch back to the `nginx` one.